db/migrate/20191227234358_add_domain_to_stories.rb

```diff
@@ -0,0 +1,5 @@
+class AddDomainToStories < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :stories, :domain, foreign_key: true
```

Comment:
`Story.all.each {|s| s.url = s.url; s.save! }`

And can just use
`%w{ 1url.com 7.ly... }.each { |d| Domain.create! domain: d, tracking: true }`